gear-test/Cargo.toml

```diff
@@ -27,6 +28,11 @@ common = { package = "gear-common", path = "../common" }
 core-processor = { package = "gear-core-processor", path = "../core-processor", default-features = false }
 gear-backend-common = { path = "../core-backend/common" }
 gear-backend-wasmtime = { path = "../core-backend/wasmtime" }
+pallet-gear = { path = "../pallets/gear" }
```

Comment:
Do we really can't avoid substrate deps in gear-test? That's horrible

Reply:
If I understand correctly the entire dependencies direction this can be done by refactor `InstrumentedCode` a bit and use https://docs.rs/wasm-instrument/0.1.1/wasm_instrument/gas_metering/struct.ConstantCostRules.html for tests.

Reply:
> If I understand correctly the entire dependencies direction this can be done by refactor `InstrumentedCode` a bit and use [docs.rs/wasm-instrument/0.1.1/wasm_instrument/gas_metering/struct.ConstantCostRules.html](https://docs.rs/wasm-instrument/0.1.1/wasm_instrument/gas_metering/struct.ConstantCostRules.html) for tests.

But we need gas rules from `Schedule` to get real gas consumption.

Reply:
Do we really need it in tests?

Reply:
> Do we really need it in tests?

@breathx what do you think?